Makefile

```diff
@@ -47,8 +47,8 @@ $(KUBERNETES_FILES):
 prepare-local-env: $(KUBERNETES_FILES) ## Prepare the local environment for running the broker locally
 	@echo > /dev/null
 
-build-image: broker ## Build a docker image with the broker binary
-	cp broker build/broker
+build-image: ## Build a docker image with the broker binary
+	env GOOS=linux go build -i -ldflags="-s -s" -o ${BUILD_DIR}/broker ./cmd/broker
```

Comment:
The only downside to this is that it will build everytime, where the previous version only did it if the broker was out of date. One could argue though you wouldn't be calling build-image if you didn't cause a change